os/mm/umm_heap/umm_calloc.c

```diff
@@ -106,12 +106,19 @@ void *calloc_at(int heap_index, size_t n, size_t elem_size)
 
 FAR void *calloc(size_t n, size_t elem_size)
 {
-	int heap_idx;
+	int heap_idx = 0;
+	int prio_idx = 0;
 	void *ret;
+
+#ifdef CONFIG_RAM_MALLOC_PRIOR_INDEX
```

Comment:
This is not a change of `external/esp_idf_port: enable spiram on ESP`. Please split platform changes.

Reply:
platform change split to #2872 PR , please review, thanks